src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator start working");

```

Comment:
The message 'Excavator start working' could be more informative. According to the checklist, it should indicate that it's an Excavator that is working. Consider rephrasing it to something like 'Excavator is starting its work' to make it clear and informative.